maintainers/maintainer-list.nix

```diff
@@ -2045,6 +2045,12 @@
     github = "ertes";
     name = "Ertugrul Söylemez";
   };
+  esclear = {
+    email = "esclear@users.noreply.github.com";
+    github = "esclear";
+    githubId = 7432848;
+    name = "Daniel Albert";
+  };
```

Comment:
can you make a separate commit out this?

Reply:
Done.